app/controllers/api/metrics_controller.rb

```diff
@@ -23,27 +23,43 @@ def query_series
                .between(from: from, to: to)
                .by_base_path(format_base_path_param)
                .run
-    if Metric.is_edition_metric?(metric)
-      series
-        .with_edition_metrics
-        .order('dimensions_dates.date asc')
-        .pluck(:date, "facts_editions.#{metric}").to_h
-    else
-      series
-        .order('dimensions_dates.date asc')
-        .pluck(:date, metric).to_h
+
+    metric_names = format_metric_names
```

Comment:
I would include this logic within `Report#Series` and would add tests accordingly